src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine{

```

Comment:
There should not be an empty line after the class declaration. Please remove the redundant empty line to comply with the coding standards.